src/dataLoaders/components/collectorsWizard/select/SelectCollectorsStep2.tsx

```diff
@@ -26,60 +26,67 @@ import {getAll} from 'src/resources/selectors'
 
 // Utils
 import {isSystemBucket} from 'src/buckets/constants'
+import {TelegrafPlugin} from 'src/types/dataLoaders'
 
 export interface OwnProps extends CollectorsStepProps {
   buckets: Bucket[]
 }
 
+interface State {
+  selectedBucketName: string
+}
+
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = OwnProps & ReduxProps
 
 @ErrorHandling
-export class SelectCollectorsStep extends PureComponent<Props> {
+export class SelectCollectorsStep extends PureComponent<Props, State> {
+  state = {selectedBucketName: ''}
   public render() {
-    const selectedBucketName = this.props.bucket ?? this.props.buckets[0]?.name
     return (
-      <Form
-        onSubmit={this.props.onIncrementCurrentStepIndex}
-        className="data-loading--form"
-      >
-        <DapperScrollbars
-          autoHide={false}
-          className="data-loading--scroll-content"
+      <>
```

Comment:
this `<>` looks unnecessary since `<Form>` is the top level component returned

Reply:
100% not needed, removed